test/e2e/util/util.go

```diff
@@ -1276,7 +1278,7 @@ func EnsurePodsWithEmptyDirPVsHaveSafeToEvictAnnotations(t *testing.T, ctx conte
 			for _, volume := range pod.Spec.Volumes {
 				// Check the pod's volumes, if they are emptyDir or hostPath,
 				// they should include that volume in the annotation
-				if volume.EmptyDir != nil || volume.HostPath != nil {
+				if (volume.EmptyDir != nil && volume.EmptyDir.Medium != corev1.StorageMediumMemory) || volume.HostPath != nil {
```

Comment:
We need to do this for AWS as well?

Reply:
yep this is a general check for local memory , see [here](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-types-of-pods-can-prevent-ca-from-removing-a-node)